_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -0,0 +1,280 @@
+
+- The language of the page can be set by other methods than the `lang` attribute, for example using HTTP headers or the `meta` element. These methods are not supported by all assistive technologies. This rule assumes that these other methods are insufficient to satisfying [Success Criterion 3.1.1: Language of Page](https://www.w3.org/TR/WCAG21/#language-of-page).
+
+- This rule assumes that user agents and assistive technologies can programmatically determine [valid language tags](#valid-language-tag) even if these do not conform to the [BCP 47][] syntax.
+
+- This rule assumes that [grandfathered tags][] are not used as these will not be recognized as [valid language tags](#valid-language-tag).
+
+- This rule assumes that `iframe` title elements are not exposed to assistive technologies and so does not consider them as part of the [default page language][].
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 3.1.1: Language of Page](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html)
+- [H57: Using language attributes on the html element](https://www.w3.org/WAI/WCAG21/Techniques/html/H57)
+- [BCP 47: Tags for Identifying Languages](https://www.ietf.org/rfc/bcp/bcp47.txt)
+- [The `lang` and `xml:lang` attributes](https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This page has a [default language][default page language] of English, which matches the language indicated in the `lang` attribute.
+
+```html
+<html lang="en">
+	<head>
+		<title>ACT Rules Format 1.0 - Abstract</title>
+	</head>
+	<body>
+		<p>
+			The Accessibility Conformance Testing (ACT) Rules Format 1.0 defines a format for writing accessibility test
+			rules. These test rules can be used for developing automated testing tools and manual testing methodologies. It
+			provides a common format that allows any party involved in accessibility testing to document and share their
+			testing procedures in a robust and understandable manner. This enables transparency and harmonization of testing
+			methods, including methods implemented by accessibility test tools.
+		</p>
+	</body>
+</html>
+```
+
+#### Passed Example 2
+
+This page has a [default language][default page language] of English because most words are English. The [default page language][] matches the language indicated in the `lang` attribute.
+
+```html
+<html lang="en">
+	<head>
+		<title>Gelukkig</title>
+	</head>
+	<body>
+		<p>The Dutch word "gelukkig" has no equivalent in English.</p>
+	</body>
+</html>
+```
+
+#### Passed Example 3
+
+This page has a [default language][default page language] of Dutch because all English text is wrapped in a `p` element with a `lang` attribute set to `en`. Dutch ("nl") is set as the language of the page by the `lang` attribute on the `html` element.
```

Comment:
Examples description varies, shouldn't it be consistent?

Reply:
I don't understand what you mean with this.